### PR TITLE
Category Normalizer Mismatch

### LINKED ISSUE #62: Query plan not contacting all available KPs
The following query graph is not contacting a KP that advertises AnatomicalEntity.
```json
{
  "message": {
    "query_graph": {
      "nodes": {
        "n0": {
          "category": "biolink:Gene"
        },
        "n1": {
          "category": "biolink:AnatomicalEntity",
          "id": "GO:0005634"
        }
      },
      "edges": {
        "e0": {
          "subject": "n0",
          "object": "n1",
          "predicate": "biolink:part_of"
        }
      }
    }
  }
}
```

### PR DESCRIPTION
**Pull request by alongreyber**
Fixed #62 where the category given does not match the one that is in the normalizer. The fix involves passing the expanded query graph into StriderWorker and then using that for partial results.

### CODE REVIEW

**Review by patrickkwang — COMMENTED** [review]
What if we move the qgraph expansion out of `generate_plans()`? It seems to me that we can basically do it as soon as we receive the qgraph, and throw away the original. This isn't really like an intermediate artifact, it's just how we're interpreting the qgraph. Would that cause problems?

**Comment by alongreyber** [comment]
I'm not sure if it will cause problems, but I will give it a try. It would make things a bit cleaner.

File: strider/server.py [hunk 1]
```diff
@@ -358,7 +358,7 @@ async def generate_traversal_plan(
 ) -> list[dict]:
     """Generate plans for traversing knowledge providers."""
     query_graph = query.message.query_graph.dict()
-    plans = await generate_plans(query_graph)
+    plans, _ = await generate_plans(query_graph)
```
Comment (patrickkwang) [inline strider/server.py:361]: I suspect this is broken. We should write tests for the actual endpoints using FastAPI's TestClient.

File: strider/query_planner.py [hunk 2]
```diff
@@ -391,18 +296,19 @@ def dfs(operation_graph, source) -> (list[str], list[str]):
 async def generate_plans(
         qgraph: QueryGraph,
         kp_registry: Registry = None,
-        normalizer: Normalizer = None,
         logger: logging.Logger = LOGGER,
-) -> list[dict[Step, list]]:
+) -> (list[dict[Step, list]], dict[str, dict]):
```
Comment (patrickkwang) [inline strider/query_planner.py:300]: This output signature needs to revert.

File: tests/server/test_server.py [hunk 3]
```diff
@@ -213,6 +213,50 @@ async def test_solve_missing_category():
...
+        """
+            CHEBI:6801(( category biolink:Drug ))
+            MONDO:0005148(( category biolink:Disease ))
+            CHEBI:6801-- predicate biolink:treats -->MONDO:0005148
+        """
+    },
+    normalizer_data="""
+        CHEBI:6801 categories biolink:Drug
+        """
+
```
Comment (patrickkwang) [inline tests/server/test_server.py:231]: Extra newline?

**Review by patrickkwang — CHANGES_REQUESTED** [review]
I like this organization much better! Minor fixes.

**Review by patrickkwang — APPROVED** [review]

**Pull request merged**